packages/shared/src/components/cards/v1/WelcomePostCard.tsx

```diff
@@ -125,7 +125,7 @@ export const WelcomePostCard = forwardRef(function SharePostCard(
               alt="Post Cover image"
               src={image}
               fallbackSrc={cloudinary.post.imageCoverPlaceholder}
-              className="my-2 object-cover"
+              className="my-2 object-cover mobileXXL:self-start mobileXXL:object-fill"
```

Comment:
Just suggestion to add this line in the component if it's being added to all components 🙏

Reply:
sry, missed your review. That's a good point, but these cards are used for the `share` and `feedback` cards as well, and we don't want these classes there 🙏